Resources/Prototypes/DeltaV/Entities/Markers/Spawners/ghost_roles.yml

```diff
@@ -17,6 +17,25 @@
         - sprite: Mobs/Species/Human/parts.rsi
           state: full
 
+- type: entity
+  parent: SpawnPointPlayerCharacter
+  id: SpawnPointPlayerCharacterMindShield
+  name: ghost role spawn point
+  suffix: mindshielded, player character
+  components:
+  - type: GhostRoleCharacterSpawner
+    mindShield: true
+
+- type: entity
+  parent: SpawnPointPlayerCharacter
+  id: SpawnPointPlayerCharacterTargetImmune
+  name: ghost role spawn point
+  suffix: mindshielded, KillObjectiveImmune, player character
+  components:
+  - type: GhostRoleCharacterSpawner
+    mindShield: true
+    targetObjectiveImmune: true
```

Comment:
```
addedComponents:
- MindShield
- AntagImmune
- TargetObjectiveImmune
```

Reply:
i wanted it to be easily VV'able but yeah real

Reply:
would be true if there werent prototypes but there are, so no reason not to do this (if you are going to VV something just use a different prototype)
this is more flexible for prototypes and will be infinitely more flexible for VV if/when list gets vv writing support

Reply:
I mean I have loadprototype and SCSI anyways and the admins can just cope so yeah